library/psa_crypto.c

```diff
@@ -3692,7 +3692,9 @@ psa_status_t psa_aead_decrypt( mbedtls_svc_key_id_t key,
     return( status );
 }
 
-static psa_status_t psa_validate_tag_length( psa_algorithm_t alg ) {
+static psa_status_t psa_validate_tag_length( psa_algorithm_t alg )
+{
+    psa_status_t status = PSA_SUCCESS;
```

Comment:
```suggestion
    psa_status_t status = PSA_ERROR_CORRUPTION_DETECTED;
```

Reply:
That won't work unless the code is changed to assign status to PSA_SUCCESS in each case below.
I went for the "least changes", but if I were to change that, I would change the code to only have one return statement, but I'm not sure if that would violate section "Exit early and prevent nesting" in the coding standard?

Reply:
Ah, I see. You could change the `break` statements to `return PSA_SUCCESS` I suppose. But I'm also fine with the "least changes" approach and avoiding scope-creep. So I'm happy to leave this as you've done it.